### PR TITLE
feat: add appendix with test cases

### PR DESCRIPTION
**Pull request by raimundo-henriques**

### CODE REVIEW

File: specs/index.bs [hunk 1]
```diff
@@ -2543,6 +2543,105 @@ A Product Footprint with a <{HOC}> highlighted:
   </pre>
 </div>
 
+# Appendix C: iLEAP Conformance Testing # {#appendix-c}
+
+In order to test the conformance of an iLEAP implementation, the following tests must be performed:
+
+## PACT Conformance Tests ## {#pact-conformance-tests}
+
+Sine the iLEAP Technical Specifications were conceived as an extension to the PACT Data Model and
```
Comment (GabrielaRubioDomingo) [inline specs/index.bs:2554]: Since

**Review by GabrielaRubioDomingo — COMMENTED** [review]
I saw a very small typo. I am unfortunately unable to do a deep review or notice any major errors, but the looks of it are good. - I understand this is about the data required and the expected responses.

File: specs/index.bs (hunk 1 above)
Comment (raimundo-henriques) [reply specs/index.bs]: Thank you!!

**Comment by raimundo-henriques** [comment]
Thank you, @GabrielaRubioDomingo !
The goal is to provide testers with a clear indication of how they should engage in bi-lateral testing. It is in need of more feedback, but we have presented it in the bi-weekly meetings before.

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Review by GabrielaRubioDomingo — APPROVED** [review]

**Pull request merged**